rvic/parameters.py

```diff
@@ -576,6 +576,11 @@ def gen_uh_run(uh_box, fdr_data, fdr_vatts, dom_data, outlet, config_dict,
     outlet.cell_id_source = dom_data['cell_ids'][y, x]
     outlet.x_source = x
     outlet.y_source = y
+
+    # Make sure the inds are all greater than zero GH79
```

Comment:
Please elaborate - shorthand that is only understood by the in crowd is simply jargon and not useful. Remember that this is not Xray, so your audience is different. If I didn't know what I meant it is likely that most other RVIC users don't either. Basically just remove it. You can include the issue number in the commit message rather than the code.